packages/neos-ui-backend-connector/src/Endpoints/index.js

```diff
@@ -121,6 +121,16 @@ export default routes => {
     })).then(response => response.json())
     .catch(reason => fetchWithErrorHandling.generalErrorHandler(reason));
 
+    const contentDimensions = (dimensionName, chosenDimensionPresets) => fetchWithErrorHandling.withCsrfToken(() => ({
+        url: urlWithParams(`${routes.core.service.contentDimensions}/${dimensionName}.json`, {chosenDimensionPresets}),
+        method: 'GET',
+        credentials: 'include',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    })).then(response => response.json())
```

Comment:
I'm just asking, but as we are on node 8.9.4 now, shouldn't we try to use async, await?

Reply:
Wait, how does it relate to node? This is Babel code executed in the browser...      
But yes, could use async here, but I'd rather be consistent and replace it everywhere, so the code would be easier to understand

Reply:
You are right, my fault.
As it runs through babel anywhere I would say let's keep it this way.